crt_portal/cts_forms/templates/partials/banner/usa_banner_language_selection.html

```diff
@@ -0,0 +1,20 @@
+{% load i18n %}
+
+{% get_current_language as LANGUAGE_CODE %}
+{% get_available_languages as LANGUAGES %}
+{% get_language_info_list for LANGUAGES as languages %}
+
+<div class="grid-col-fill language-selection">
+    <div class="grid-col-auto language-selection__inner">
+      <img src="/static/img/globe-language-white.svg" alt="language" class="icon">
```

Comment:
Good spot to use the `static` template tag instead of hard coding the path to this `.svg`